plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -825,27 +845,41 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
     rotationDegreesSinceLast: Float
   ): Boolean {
     // in order not to mess up initial anchor values
-    if (immediateEaseInProcess) {
+    if (internalSettings.disableRotateWhenPinchingToZoom && immediateEaseInProcess) {
       return true
     }
     // Calculate map bearing value
     val currentBearing = mapCameraManagerDelegate.cameraState.bearing
     rotateCachedAnchor = cameraAnimationsPlugin.anchor
     val bearing = currentBearing + rotationDegreesSinceLast
 
-    // Rotate the map
-    val focalPoint = getRotateFocalPoint(detector)
-    easeToImmediately(
-      CameraOptions.Builder()
-        .anchor(focalPoint)
-        .bearing(bearing)
-        .build(),
-      actionAfter = {
-        cameraAnimationsPlugin.anchor = rotateCachedAnchor
-        notifyOnRotateListeners(detector)
-      }
-    )
-
+    if (internalSettings.disableRotateWhenPinchingToZoom) {
+      // Rotate the map
+      val focalPoint = getRotateFocalPoint(detector)
+      easeToImmediately(
+        CameraOptions.Builder()
+          .anchor(focalPoint)
+          .bearing(bearing)
+          .build(),
+        actionAfter = {
+          cameraAnimationsPlugin.anchor = rotateCachedAnchor
+          notifyOnRotateListeners(detector)
+        }
+      )
+    } else {
+      val bearingAnimator =
+        cameraAnimationsPlugin.createBearingAnimator(cameraAnimatorOptions(bearing)) {
+          duration = 0
+          interpolator = AccelerateDecelerateInterpolator()
```

Comment:
why this interpolator? if it's used anywhere else please move it to val or const

Reply:
This could be removed.